Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3
+FROM python:3-alpine
```

Comment:
@pb82 It might be best to go with a centos base image here to minimise changes when running on rhel (downstream)?